include/VCF_Globals.h

```diff
@@ -17,9 +20,10 @@
 #include <etl/singleton.h>
 
 /* Interface and system data structs */
-extern VCFInterfaceData_s interface_data; // NOLINT
-extern VCFSystemData_s system_data; // NOLINT
-extern VCRSystemData_s vcr_system_data; // NOLINT
+extern VCFData_s vcf_data; // NOLINT
+extern VCFInterfaceData_s interface_data;
+extern VCFSystemData_s system_data;
+extern VCRSystemData_s vcr_system_data;
```

Comment:
VCF Interface data and System data are nexted within VCFData-- so you don't really need separate instances of interface/system data.

Reply:
Also, you can either have VCRSystemData or just VCRData, doesn't really matter. up to you!